src/main_menu.cpp

```diff
@@ -243,6 +242,55 @@ std::vector<std::string> main_menu::load_file( const std::string &path,
     return result;
 }
 
+holiday main_menu:: get_holiday_from_time()
+{
+    std::tm *local_time;
+
+    std::time_t current_time = std::time( nullptr );
+    local_time = std::localtime( &current_time );
```

Comment:
In principle `localtime` can fail and return `nullptr`.  You might want to handle that.  Not sure if it really happens in practice.

Reply:
Thanks, the code now checks localtime's return.